dbase/table.go

```diff
@@ -48,19 +36,37 @@ type Column struct {
 	Reserved   [8]byte  // Reserved
 }
 
-// ColumnModification contains the modification to change values or name of columns
-type ColumnModification struct {
-	Trimspaces  bool
-	Convert     func(interface{}) interface{}
-	ExternalKey string
+// Table is a struct containing the table columns, modifications and the row pointer
+type Table struct {
+	// Columns defined in this table
+	columns []*Column
+	// Modification to change values or name of fields
+	mods []*Modification
+	// Internal row pointer, can be moved
+	rowPointer uint32
```

Comment:
Tables have a row limit? uint32 -> 64

Reply:
> # Table Header Record Structure 
> 
> <!--StartFragment-->
> 
> Byte offset | Description
> -- | --
> 0 | File type: 0x01FoxBASE: 0x02FoxBASE+/Dbase III plus, no memo: 0x2FVisual FoxPro: 0x30Visual FoxPro, autoincrement enabled: 0x31Visual FoxPro, Varchar, Varbinary, or Blob-enabled: 0x42dBASE IV SQL table files, no memo: 0x62dBASE IV SQL system files, no memo: 0x82FoxBASE+/dBASE III PLUS, with memo: 0x8AdBASE IV with memo: 0xCAdBASE IV SQL table files, with memo: 0xF4FoxPro 2.x (or earlier) with memo: 0xFA
> 1 - 3 | Last update (YYMMDD)
> 4 – 7 | Number of records in file
> 
> <!--EndFragment-->
> 

**The number of records in the file header is defined by Microsoft as uint32.**